### ISSUE TITLE
make the UI responsive and pretty it up a bit

### ISSUE THREAD

**Issue by bamohan** [katharinap/my_recipes#58]
On Edit page, make the following responsive
- [ ] - file picker for images 
- [x]  - remove button

**Comment by bamohan** [comment]
The issue with file Picker requires a change to these 2 files
- vendor/assets/javascripts/bootstrap-filestyle.js
- vendor/assets/javascripts/bootstrap-filestyle.min.js

The change is to change `<div class="bootstrap-filestyle input-group">'` to  `<div class="bootstrap-filestyle">'` i.e.remove `input-group` from class

This makes the UI responsive. However, doesn't look great on desktop. so need to decide whether to do that

**Comment by katharinap** [comment]
I made an attempt to change the style of the page because I'm not super happy with how it looks anymore.  I created a temp branch, viewtest, for this because I was only playing around with things.  If we want to go this direction, I'd like to start over again with the changes because it's quite messy now.

If you have time to test it out, you will need to create new versions of images in medium size.  This can be done with

Recipe.find_each do |recipe|
  recipe.picture.recreate_versions!(:mid) if recipe.picture?
end 
 
If you do not have any pictures assigned to any of your recipes yet, it is going to use a default image.

This is far from a final pretty solution but it might be a start.  I also added a recipe search.
